dev/com.ibm.ws.security.jwt/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -106,4 +106,19 @@ elapsedNBF.desc=Identifies the elapsed time until JWT will start to be accepted
 
 #Do not translate "amr"
 amrValues=Authentication methods references in string array
-amrValues.desc= Specifies Authentication Methods References so it can enforce step-up authentication and know how the user is authenticated. Specify a comma separated list of valid amr claim values to be used.
\ No newline at end of file
+amrValues.desc= Specifies Authentication Methods References so it can enforce step-up authentication and know how the user is authenticated. Specify a comma separated list of valid amr claim values to be used.
+
+# Do not translate "Content Encryption Key"
+keyManagementKeyAlgorithm=Key management key algorithm
+keyManagementKeyAlgorithm.desc=Specifies the encryption algorithm that is used to encrypt the Content Encryption Key of a JWE.
+
+keyManagementKeyAlgorithm.RSA-OAEP=Use the RSAES OAEP algorithm to encrypt the Content Encryption Key of a JWE.
+
+keyManagementKeyAlias=Key management key alias
+keyManagementKeyAlias.desc=Public key alias of the Key Management Key that is used to encrypt the Content Encryption Key of a JWE.
```

Comment:
- content encryption 
- key management key

Reply:
Should this be Key Management Mode? I see "Key Management Mode" defined in the spec but not Key Management Key. (https://tools.ietf.org/html/rfc7516#section-2)

If it should remain Key Management Key- the capitalization  should be consistent between the name and description

Reply:
I decided to lowercase the phrase since it doesn't refer to a well-defined concept from the spec.